sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/ier12.processing.service.ts

```diff
@@ -134,37 +144,168 @@ export class IER12ProcessingService {
    * @param pendingAssessment pending assessment of institutions.
    * @returns IER 12 record.
    */
-  private createIER12Record(pendingAssessment: StudentAssessment): IER12Record {
+  private async createIER12Record(
+    pendingAssessment: StudentAssessment,
+  ): Promise<IER12Record[]> {
```

Comment:
maybe a small refactor for the return comment

Reply:
thanks for mentioning it.